src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_safeguards.py

```diff
@@ -58,24 +58,24 @@ def test_aks_deployment_safeguards(self):
 
         # Enable safeguards
         self.cmd(f'aks safeguards create -c {aks_cluster["id"]} --level Warn', checks=[
-            self.check('properties.level', 'Warn'),
+            self.check('level', 'Warn'),
```

Comment:
This appears to be a breaking change. Could you please double-check if this is intentional? Breaking changes are only permitted twice a year, and the most recent allowed window has just passed. The next opportunity will be in May next year.

Reply:
Reverting